packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -84,18 +84,18 @@ const Updates = decorate([
   ]),
   provideState({
     initialState: () => ({
-      _channel: undefined,
       ...initialProxyState(),
       ...initialRegistrationState(),
       askRegisterAgain: false,
+      channel: undefined,
       showPackagesList: false,
     }),
     effects: {
       async configure() {
         const { effects, state } = this
         await xoaUpdater.configure({
           ...pick(state, [
-            'channel',
+            state.channel && 'channel',
```

Comment:
The test is not necessary (and not used for other entries, please follow the existing style).